app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,8 @@
+class SessionsController < ApplicationController
+  def create
+    user = UsersFacade.login_user(request.env['omniauth.auth'])
```

Comment:
find_or_create_from_auth_hash method? it's in the docs but you might have tried that

from docs:
def create
    @user = User.find_or_create_from_auth_hash(auth_hash)
    self.current_user = @user
    redirect_to '/dashboard'
  end

Hard to know what y'all have tried and what's working for you :)

Reply:
also just saw this too for above the create method from the docs: 
skip_before_action :verify_authenticity_token, only: :create

Reply:
I think this would only work if we have the user database in the frontend. Since we're having to make a post call to the BE, not sure it would do the same thing.

Reply:
mmm I take this back because you're doing this in the Users Facade connecting to the back end - my bad!